druid/src/main/java/org/apache/calcite/adapter/druid/DruidJsonFilter.java

```diff
@@ -268,14 +268,11 @@ abstract class DruidJsonFilter implements DruidJson {
 
   private static @Nullable DruidJsonFilter toInKindDruidFilter(RexNode e, RelDataType rowType,
       DruidQuery druidQuery) {
-    switch (e.getKind()) {
```

Comment:
This seems an equivalent rewrite? In that case please revert, we try to keep the diff as minimal as possible unless there is a compelling reason for the change.

Reply:
Had changed here as before.